modules/formulize/include/entriesdisplay.php

```diff
@@ -4116,12 +4100,12 @@ function formulize_gatherDataSet($settings, $searches, $sort, $order, $frid, $fi
 		print $GLOBALS['xoopsSecurity']->getTokenHTML();
 	}
 
+	$formulize_cachedDataId = null;
```

Comment:
@tikyon This variable was not set previously. It is now set, so PHP is happier. But since it had no value, nothing would be sent in $_POST['formulize_cachedata']. That is relevant to advanced_calculations (Procedures) because in the class/advancedCalculation.php file, this value is picked up from POST and used to determine the file name of some cached data. Maybe this should have a value?? Except it hasn't for ages and ages and everything is OK? Since you guys work with the procedures more than anyone, your opinion really matters here.